src/Sylius/Behat/Service/Provider/Calendar.php

```diff
@@ -14,7 +14,7 @@
 namespace Sylius\Behat\Service\Provider;
 
 use Sylius\Bundle\ShippingBundle\Provider\DateTimeProvider;
-use Sylius\Component\Promotion\Provider\DateTimeProviderInterface;
+use Sylius\Calendar\Provider\DateTimeProviderInterface;
 
 final class Calendar implements DateTimeProvider, DateTimeProviderInterface
```

Comment:
The idea of extracting this component was partially to get rid of this class 😄 so we should remove it as it's not necessary anymore, we can use [FakeCalendar](https://github.com/Sylius/Calendar/blob/master/tests/Behat/Provider/FakeCalendar.php) class from the new component